README.md

```diff
@@ -43,7 +43,7 @@ class ProductVariables(BaseVariables):
         last_order = self.product.orders[-1]
         return (last_order.expiration_date - datetime.date.today()).days
 
-    @string_rule_variable(cache_result=False)
+    @string_rule_variable
```

Comment:
I think the parens are still necessary for this decorator (it's not compatible with both styles of calling).